### PR TITLE
chore: add integration dbm tests

### PR DESCRIPTION
**Pull request by majorgreys**
- Adds postgres to docker services
- Adds `/dbm` route to flask poc app
- Adds validator for python traces
   - Validates that database monitoring span tag (`_dd.dbm_trace_injected`) is set when postgres queries are executed.

### CODE REVIEW

File: run.sh [hunk 1]
```diff
@@ -136,6 +136,8 @@ elif [ $SYSTEMTESTS_SCENARIO = "REMOTE_CONFIG_MOCKED_BACKEND_ASM_DD_NOCACHE" ];
 
 elif [ $SYSTEMTESTS_SCENARIO = "TRACE_PROPAGATION_STYLE_W3C" ]; then
     WEBLOG_ENV+="DD_TRACE_PROPAGATION_STYLE_INJECT=W3C\nDD_TRACE_PROPAGATION_STYLE_EXTRACT=W3C"
```
Comment (mabdinur) [inline run.sh:138]: create new scenario

**Comment by cbeauchesne** [comment]
This will be probably  useful : https://github.com/DataDog/system-tests/blob/main/docs/edit/lint.md

File: utils/build/docker/python/flask-poc.Dockerfile [hunk 2]
```diff
@@ -4,7 +4,7 @@ FROM python:3.9
 RUN python --version && curl --version
 
 # install hello world app
-RUN pip install flask gunicorn gevent requests
+RUN pip install flask gunicorn gevent requests psycopg2
```
Comment (cbeauchesne) [inline utils/build/docker/python/flask-poc.Dockerfile:7]: you probably need to add this on uwsgi.Dockerfile, as it uses flask file

**Review by cbeauchesne — CHANGES_REQUESTED** [review]

**Review by mabdinur — APPROVED** [review]

File: docker-compose.yml [hunk 3]
```diff
@@ -53,6 +53,8 @@ services:
         condition: service_healthy
       agent:
         condition: service_healthy
+      postgres:
+        condition: service_started
```
Comment (cbeauchesne) [inline docker-compose.yml:57]: There us an healthcheck on postgres, is there a reason not using  `service_healthy` ?

File: utils/build/docker/python/flask.py [hunk 4]
```diff
@@ -104,6 +108,24 @@ def identify_propagate():
     return Response("OK")
 
 
+@app.route("/dbm")
+def dbm():
```
Comment (cbeauchesne) [inline utils/build/docker/python/flask.py:112]: can you add in https://github.com/DataDog/system-tests/tree/main/docs/weblog the expected behavior of this endpoint ?

**Review by cbeauchesne — CHANGES_REQUESTED** [review]

File: docker-compose.yml (hunk 3 above)
Comment (mabdinur) [reply docker-compose.yml:57]: Nope. I think this was set during testing to debug issues. I'll revert this

**Review by cbeauchesne — APPROVED** [review]

**Pull request merged**